src/constants.ts

```diff
@@ -94,4 +95,10 @@ export const relationships: Record<
     sourceType: entities.USER._type,
     targetType: entities.TOKEN._type,
   },
+  TOKEN_ALLOWS_VENDOR: {
+    _type: 'google_token_allows_mapped_vendor',
+    _class: RelationshipClass.ALLOWS,
+    sourceType: entities.TOKEN._type,
+    targetType: 'mapped_entity (class Vendor)',
```

Comment:
We haven't had this come up often (maybe at all?) yet, but when documentation should be created for a relationship where the actual `_type` for either `sourceType` or `targetType` is unknown, I think we should differentiate from explicit `_type`s. Since `_type`s use the `lower_snake_case` convention, maybe we could use `UPPER_SNAKE_CASE`  so that documentation easily distinguishes where our relationships are to a specific `_type` or some unknown `_type`.

```suggestion
  TOKEN_ALLOWS_VENDOR: {
    _type: 'google_token_allows_mapped_vendor',
    _class: RelationshipClass.ALLOWS,
    sourceType: entities.TOKEN._type,
    targetType: 'MAPPED_VENDOR',
```

Reply:
@ndowmon Thanks! I did think a bit about tests. Note: existing tests pass before and after this change. This project doesn't yet make use of the recording setup that newer projects do, and that seemed out of scope of this change. I did verify functionality through several job invocations in dev.

Reply:
Re: UPPER_SNAKE_CASE: maybe that could be a thing? In earlier conversation with @aiwilliams I mentioned that I'd like the auto-generated documentation to also include source and target Classes, which would make more explicit what's happening here, without abusing the notion of `_type` like I'm forced to currently. Showing the source Class would also help folks learn about how this integration interactions with our Abstract Data Model.

Reply:
Would you mind opening an issue in the SDK repo with a `documentation` label, that links back to this PR?

Reply:
Opened https://github.com/JupiterOne/sdk/issues/357